docs/docs/index.md

```diff
@@ -293,52 +275,14 @@ output if {
 
 <RunSnippet files="#input.json" command="data.servers.output"/>
 
-Variables are immutable. OPA reports an error if you try to assign the same
```

Comment:
I mention they're immutable above now, and felt this was too much detail for the homepage.